devfile.yaml

```diff
@@ -22,48 +25,7 @@ components:
   - name: outerloop-deploy
     kubernetes:
       uri: outerloop-deploy.yaml
-  - name: runtime
-    container:
```

Comment:
@yangcao77 - could you link me to the location where this configuration is now stored/what replaces it? I'm curious to see where we would need to update things like the Node.js version.